e2e/Dockerfile

```diff
@@ -10,6 +10,13 @@ RUN wget -q https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_
     wget -q https://get.helm.sh/helm-${HELM_VERSION}-linux-amd64.tar.gz -O - | tar -xzO linux-amd64/helm > /usr/local/bin/helm && \
     chmod +x /usr/local/bin/helm
 
+WORKDIR /usr/src/app
+COPY go.mod go.sum ./
+RUN go mod download && go mod verify
+COPY . .
```

Comment:
sonarcloud complains about this one for a good reason, can we make this explicit :pray: ?

Reply:
I actually think this usage here is fine.

Sonar complains with "Copying recursively might inadvertently add sensitive data to the container. Make sure it is safe here". The copying happens in the builder container though, which is only used instead of the host machine to compile binaries. The builder image is thrown away in the build process, and only specific artifacts (the same ones that were copied before) are copied to the final alpine-based image. The security posture of the final image would not improve by copying each folder (like `suites`) individually.

Do you agree with this reasoning? If so, do you know how to silence SonarQube? I guess adding some comment above this line along the lines of `# disable docker:S6470`? I know that in e.g. Java one can do `@SurpressWarnings("some:rule")` but I did not find documentation how to do it in Dockerfiles.